man/aotprof-tool.1

```diff
@@ -0,0 +1,91 @@
+.TH aotprof-tool 1
```

Comment:
Should a tool's name include the -tool suffix ?

Reply:
It should not, @radekdoulik could come up with a better name?